### PR TITLE
Updated kbcli to 0.24.0

### PR DESCRIPTION
**Pull request by KBbitsP**
Hi @pierre @reshmabidikar 

Please review

### CODE REVIEW

File: kbclient/account/account_client.go [hunk 1]
```diff
@@ -6,37 +6,15 @@ package account
...
-	defaults KillbillDefaults) *Client {
-
-	return &Client{transport: transport, formats: formats, authInfo: authInfo, defaults: defaults}
-}
-
-// killbill default values. When a call is made to an operation, these values are used
-// if params doesn't specify them.
-type KillbillDefaults interface {
-	// Default CreatedBy. If not set explicitly in params, this will be used.
-	XKillbillCreatedBy() *string
```
Comment (pierre) [inline kbclient/account/account_client.go:31]: I think the custom templates weren't taken into account, as this section of the code is missing now: https://github.com/killbill/go-swagger/blob/killbill/generator/templates/client/client.gotmpl#L41

Maybe the step 1 from the README failed?

```
go1.13 generate ./generator
```

Until we have CI for this, the easiest way to verify the new build would be to run the `kbcmd` tutorial from https://github.com/killbill/kbcli/blob/master/kbcmd/README.md

**Review by pierre — CHANGES_REQUESTED** [review]

File: kbclient/account/account_client.go (hunk 1 above)
Comment (sbrossie) [reply kbclient/account/account_client.go:31]: FYI, filed:
* https://github.com/killbill/kbcli/issues/29
* https://github.com/killbill/kbcli/issues/30
* https://github.com/killbill/kbcli/issues/31

Comment (KBbitsP) [reply kbclient/account/account_client.go]: Hi @pierre,

I have regenerated the client on linux machine. Please review if it looks ok now.

Updated README.md too.

Comment (KBbitsP) [reply kbclient/account/account_client.go]: Hi @pierre could you please review the latest changes. 

adding @reshmabidikar

**Review by pierre — APPROVED** [review]
Looks much better!

**Pull request merged**